src/components/OcStatusIndicators.vue

```diff
@@ -72,6 +82,17 @@ export default {
     hasHandler(indicator) {
       return Object.prototype.hasOwnProperty.call(indicator, "handler")
     },
+    getIndicatorDescriptionId(indicator) {
+      if (!indicator.accessibleDescription) {
+        return null
+      }
+
+      if (!indicator.accessibleDescriptionId) {
```

Comment:
What is the use case for providing an `accessibleDescriptionId` via data? Since it's always created within this component it would be fine by me to always create a random one, as long as it's unique.

Reply:
Maybe I get it wrong, but each `indicator` has its own `accessibleDescriptionId` which needs to be "bound" to it. Hence we cannot provide a "global" unique id here.

Reply:
What I meant is that I don't see a reason why we would provide an element id from external, when the html is always coming from within this component (the span with the `accessibleDescription` that you introduced in this PR).

Reply:
`accessibleDescriptionId` is not supposed to be provided external (although it can, technically). The only reason we attach it to `indicator` is because we need to store the id somewhere. The if-condition does that initially for each `indicator`. The next time we call that method for the exact same `indicator`, it may already have an id. Then we just return it.

Reply:
Ah, now I get it. Thanks. In that case I'd prefer to cache the id in the data object of this component.

```
data() {
  return {
    accessibleDescriptionIds: {}
  }
}
```
And then instead of modifying `indicator.accessibleDescriptionId`, set `this.accessibleDescriptionIds[indicator.id] = uniqueId("oc-indicator-description-")` and reuse that in subsequent calls.

Reply:
Oh that's nice, and should make it more clear also 👍  Thanks!